src/impl_/pymethods.rs

```diff
@@ -490,6 +492,12 @@ impl<'a, 'py> BoundRef<'a, 'py, PyAny> {
         Bound::ref_from_ptr_or_opt(py, ptr).as_ref().map(BoundRef)
     }
 
+    pub unsafe fn downcast<T: PyTypeCheck>(
```

Comment:
I'd think this one is actually safe, given it's checked?

```suggestion
    pub fn downcast<T: PyTypeCheck>(
```

Reply:
Your completely right, one of those copy pasta errors 😆 . I'll fix it